README.rst

```diff
@@ -0,0 +1,154 @@
+tedana: TE Dependent ANAlysis
+=============================
+
+The ``tedana`` package is part of the ME-ICA pipeline, performing TE-dependent
+analysis of multi-echo functional magnetic resonance imaging (fMRI) data.
+``TE``-``de``\pendent ``ana``\lysis (_tedana_) is a Python module for denoising
```

Comment:
I think the underscores around `_tedana_` will need to be converted to asteriks (i.e., `*tedana*`) to render properly in rst

Reply:
👍